Dockerfile.mockbuild

```diff
@@ -13,7 +13,7 @@ ARG UID=1000
 
 # Install basic tools
 RUN yum install -y mock make rpm-build curl createrepo rpmlint redhat-lsb-core \
-                   git python-srpm-macros
+                   git python-srpm-macros rpmdevtools
 
```

Comment:
Since this is Fedora, should dnf be used instead of yum?

Reply:
Fixed in the master merge.